screens/category.css

```diff
@@ -0,0 +1,35 @@
+
+.category-container{
+    min-height: 80vh;
+    height: auto;
+    width: 90%;
+    padding: 1rem;
+    border: 1px solid var(--secondary-grey);
+    border-radius: 10px;
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    flex-wrap: wrap;
+}
+
+.category-card{
+    width: 250px;
+    height: 420px;
+    border: 1px solid var(--secondary-grey);
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+}
+
+.category-img{
+    width: 100%;
+    height: 220px;
+    object-fit: cover;
```

Comment:
Suggestion: If possible use rem here, by the way this is also fine, it is your choice